src/util/idle.ts

```diff
@@ -11,7 +11,7 @@ interface IdleHandlers {
 }
 
 export class IdleManager {
-    public static DEF_INTERVAL = 60
+    public static DEF_INTERVAL = 20
```

Comment:
@poltak I tried to send this interval in the constructor but wasn't able to get executed, so I made changes here globally.